src/amo/reducers/reviews.js

```diff
@@ -576,6 +577,12 @@ export default function reviewsReducer(
       }
       return newState;
     }
+    case LOCATION_CHANGE: {
+      return {
+        ...state,
+        view: {},
```

Comment:
I'm looking at the side effects of this. I guess one is that the app will forget which reviews you had flagged (for spam, etc) as you navigate between listing pages. That's probably fine, though.

Reply:
Thanks @kumar303. That is a bit of a pain, although it matches the behaviour if you reload the site as well (i.e., you don't see that reviews you've previously flagged are flagged unless you're in the same redux session), so I guess it's not too bad.

I could change the logic to keep the flagged view state intact and only reset the other data, but it would mean looping through all of the existing reviewIds. It sounds like you don't favour that over the current approach, but I just thought I'd check. Which do you prefer?

Reply:
I prefer your approach, no need to change it. Yeah, you could write some complex code to iterate through the entire view map and only reset `editingReview: false` but I don't think it's worth the complexity.